conda_libmamba_solver/mamba_utils.py

```diff
@@ -145,6 +145,8 @@ def init_api_context() -> api.Context:
         _get_base_url(x.url(with_credentials=True)) for x in context.default_channels
     ]
 
+    api_ctx.conda_build_local_paths = list(context.conda_build_local_paths)
```

Comment:
I was a little confused about the upstream implementation for this, but @isuruf made me aware of how this is used/overridden here.

This LGTM and would now just need a dependency adjustments to require `libmambapy>=1.5.6` and be good to go, right?

Reply:
We can also check with hasattr and do not require any newer versions. It'll take a bit for defaults to build 1.5.6 now during the break and I'll be on PTO til Jan 1st.

Reply:
Requiring 1.5.6 as of last commits!